benchmark/arrayops/arrayops.d

```diff
@@ -0,0 +1,147 @@
+/**
+ * Benchmark for array ops.
+ *
+ * Copyright: Copyright Martin Nowak 2016 -.
+ * License:   $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Authors:    Martin Nowak
```

Comment:
It's just copied from the other benchmarks, and I won't do this differently only in a single file.
Also an almost OT review comment while we're still discussing the actual implementation.